sct.py

```diff
@@ -1706,6 +1708,28 @@ def configure_jenkins_builders(cloud_provider, regions):
             raise NotImplementedError("configure_jenkins_builders doesn't support Azure yet")
 
 
+@cli.command("nemesis-list", help="Initialize an argus test run.")
+@click.option('-b', '--backend', type=click.Choice(SCTConfiguration.available_backends), help="Backend to use")
+@click.option('-c', '--config', multiple=True, type=click.Path(exists=True), help="Test config .yaml to use, can have multiple of those")
+def get_nemesis_list(backend, config):
+    add_file_logger()
+    logging.basicConfig(level=logging.WARNING)
+
+    if config:
+        os.environ['SCT_CONFIG_FILES'] = str(list(config))
```

Comment:
Are you sure this parsing works for the supported variants of that config option values?

Reply:
it was copied from `run_test`, if it would work, we would know about it

Reply:
```
$ tox -e py310 -- `pwd`/sct.py nemesis-list -b aws -c 'test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml test-cases/enterprise-features/ics/longevity/ics-longevity-toggle-strategy-large-partitions-24h.yaml'
...
py310: commands[2]> mkdir -p /home/kiparis/projects/scylla-cluster-tests/.tox/py310/lib/python3.10/site-packages
py310: commands[3]> bash -c '/home/kiparis/projects/scylla-cluster-tests/sct.py nemesis-list -b aws -c '"'"'test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml test-cases/enterprise-features/ics/longevity/ics-longevity-toggle-strategy-large-partitions-24h.yaml'"'"''
/home/kiparis/projects/scylla-cluster-tests/.tox/py310/lib/python3.10/site-packages/elasticsearch/connection/http_urllib3.py:209: UserWarning: Connecting to https://746f0ad652a3447d83b1572f657c67cb.us-east-1.aws.found.io:9243 using SSL with verify_certs=False is insecure.
  warnings.warn(
< t:2024-01-03 13:04:40,821 f:sct.py          l:186  c:root                 p:INFO  > install-bash-completion current path: /home/kiparis/projects/scylla-cluster-tests
install-bash-completion current path: /home/kiparis/projects/scylla-cluster-tests
Usage: hydra nemesis-list [OPTIONS]
Try 'hydra nemesis-list --help' for help.

Error: Invalid value for '-c' / '--config': Path 'test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml test-cases/enterprise-features/ics/longevity/ics-longevity-toggle-strategy-large-partitions-24h.yaml' does not exist.
py310: exit 2 (3.68 seconds) /home/kiparis/projects/scylla-cluster-tests> bash -c '/home/kiparis/projects/scylla-cluster-tests/sct.py nemesis-list -b aws -c '"'"'test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml test-cases/enterprise-features/ics/longevity/ics-longevity-toggle-strategy-large-partitions-24h.yaml'"'"'' pid=2756574
  py310: FAIL code 2 (12.23=setup[0.03]+cmd[3.41,5.11,0.00,3.68] seconds)
  evaluation failed :( (12.27 seconds)


$ tox -e py310 -- `pwd`/sct.py nemesis-list -b aws -c 'test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml,test-cases/enterprise-features/ics/longevity/ics-longevity-toggle-strategy-large-partitions-24h.yaml'
...
py310: commands[2]> mkdir -p /home/kiparis/projects/scylla-cluster-tests/.tox/py310/lib/python3.10/site-packages
py310: commands[3]> bash -c '/home/kiparis/projects/scylla-cluster-tests/sct.py nemesis-list -b aws -c test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml,test-cases/enterprise-features/ics/longevity/ics-longevity-toggle-strategy-large-partitions-24h.yaml'
/home/kiparis/projects/scylla-cluster-tests/.tox/py310/lib/python3.10/site-packages/elasticsearch/connection/http_urllib3.py:209: UserWarning: Connecting to https://746f0ad652a3447d83b1572f657c67cb.us-east-1.aws.found.io:9243 using SSL with verify_certs=False is insecure.
  warnings.warn(
< t:2024-01-03 13:05:01,997 f:sct.py          l:186  c:root                 p:INFO  > install-bash-completion current path: /home/kiparis/projects/scylla-cluster-tests
install-bash-completion current path: /home/kiparis/projects/scylla-cluster-tests
Usage: hydra nemesis-list [OPTIONS]
Try 'hydra nemesis-list --help' for help.

Error: Invalid value for '-c' / '--config': Path 'test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml,test-cases/enterprise-features/ics/longevity/ics-longevity-toggle-strategy-large-partitions-24h.yaml' does not exist.
py310: exit 2 (3.79 seconds) /home/kiparis/projects/scylla-cluster-tests> bash -c '/home/kiparis/projects/scylla-cluster-tests/sct.py nemesis-list -b aws -c test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml,test-cases/enterprise-features/ics/longevity/ics-longevity-toggle-strategy-large-partitions-24h.yaml' pid=2756826
  py310: FAIL code 2 (12.11=setup[0.04]+cmd[3.29,4.98,0.00,3.79] seconds)
  evaluation failed :( (12.15 seconds)

```

So, how multiple config files should really be specified?

Reply:
It's  multiple parameters you need to define it multiple times

And yeah on the pipelines we use the environment variables always not the commandline parameters, and this command would work the same

Reply:
It's  multiple parameters you need to define it multiple times

And yeah on the pipelines we use the environment variables always not the commandline parameters, and this command would work the same

Reply:
Ok, multipe `-c` works.
Would be really useful to print out the config options that were applied.
It will allow to minimize the possible bugs.

Reply:
add print of the config files used
and add few examples on the docstring of this command of different usage options